app/src/ccdashboard/client/core.cljs

```diff
@@ -7,6 +7,8 @@
     [goog.events :as events]
     cljsjs.react
     cljsjs.react-select
+    [cljsjs.d3]
```

Comment:
nitpicking: don't need the `[``]` here if you don't `:refer` or create an alias.